materialize-google-pubsub/.snapshots/TestSpec

```diff
@@ -95,11 +95,18 @@
         "type": "string",
         "title": "Resource Binding Identifier",
         "description": "Optional identifier for the resource binding. Each binding must have a unique topic \u0026 identifier pair. Included as \"identifier\" attribute in published messages if specified."
+      },
+      "create_default_subscription": {
+        "type": "boolean",
+        "title": "Create with Default Subscription",
+        "description": "Create a default subscription when creating the topic. Will be created as \"topic-sub\". Has no effect if the topic already exists.",
```

Comment:
nit: maybe consider somehow making it clear that `topic` is a variable here, e.g. `<topic>-sub`?

Reply:
That seems like a good idea. I was waffling on the description a bit here myself so the clarification should be helpful.